.gitignore

```diff
@@ -6,3 +6,4 @@ build
 apps/chat/src/Config.js
 apps/chat/src/backend/serverless/appconfig.json
 apps/chat/src/backend/serverless/.aws-sam
+ada*
```

Comment:
what is this :p Is this internal stuff :p

Reply:
Yeah when you run the script (internal) to get credentials locally a log file is added, ignored that.